CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+#### 🐛 Type: Bug
+
+- `application-shell`, `permissions`
+  - [#1009](https://github.com/commercetools/merchant-center-application-kit/pull/1009) CON-187: properly pass dataFences from menu to `Authorized.js` ([@amine-benselim](https://github.com/amine-benselim))
```

Comment:
I would avoid having ticket numbers on public repos. I can deal with it in PRs but not really in release notes :)